sdk/search/Azure.Search.Documents/src/Indexes/Models/BlobExtractionMode.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Search.Documents.Indexes.Models
+{
+    /// <summary>
+    /// Defines which parts of a blob will be indexed by the blob storage indexer.
+    /// See <see href="https://docs.microsoft.com/azure/search/search-howto-indexing-azure-blob-storage" /> for details.
+    /// </summary>
+    public readonly partial struct BlobExtractionMode : IEquatable<BlobExtractionMode>
```

Comment:
Why isn't this in the swagger instead?

Reply:
+1: this seems like it should be in the swagger